variables.tf

```diff
@@ -71,6 +71,24 @@ variable "machine_zone" {
   default     = "us-central1-a"
 }
 
+variable "machine_region" {
+  description = "The region for the machine instance to be deployed in"
+  type        = string
+  default     = "us-central1"
+}
+
+variable "machine_subnetwork" {
+  description = "The instance subnetwork name (subnet name)"
```

Comment:
Please add to the description: `the network name is not required and will be inferred from the subnetwork name.`